src/blocks/content-helper/content-helper-provider.ts

```diff
@@ -5,9 +5,6 @@ import { __ } from '@wordpress/i18n';
 import { select } from '@wordpress/data';
 // eslint-disable-next-line import/named
 import { Schema } from '@wordpress/core-data';
-import Post = Schema.Post;
-import Taxonomy = Schema.Taxonomy;
-import User = Schema.User;
```

Comment:
I'm surprised this syntax worked :)

Reply:
This was how the initial code was and it was working :)

As far as I can remember, the problems started showing up when trying to write tests using the React Testing Library.